AutoRunCustomTool/AutoRunCustomToolPackage.cs

```diff
@@ -102,12 +110,23 @@ void DocumentEvents_DocumentSaved(Document doc)
                     return;
                 targets.AddRange(runCustomToolOn.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries));
             }
+            else
+            {
+                string curExtension = "*" + Path.GetExtension(docFullPath).ToLower();
+                HashSet<string> extensions = new HashSet<string>(page.ListenToExtension.ToLower().Split(';'));
```

Comment:
There's no benefit in creating a hashset for this. It would be useful if you were going to call `Contains` many times on it, but here you're only doing it once.

Reply:
Ah, yes. I would expect it to have very few entries. Maybe at most two. Will change it.